README.md

```diff
@@ -1,70 +1,176 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] According to the project requirements for teams of 4 people:

![image](https://github.com/huzaifakhan0308/full-stack-capstone-front-end/assets/57421823/0ab5d46e-78dd-4413-ac70-b065e3b6e233)

I can't find a way or a link to delete rooms or hotels.  Kindly, add this functionality in order to meet this requirement. 👍